pipelines/skylab/slideseq/SlideSeq.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 3.1.1
```

Comment:
@ekiernan Would you mind updating the Slide-seq pipeline version in the overview doc?